cmake/modules/FindTagLib.cmake

```diff
@@ -47,19 +47,23 @@ include(IsStaticLibrary)
 
 find_package(PkgConfig QUIET)
 if(PkgConfig_FOUND)
+  if(UNIX AND NOT APPLE)
+    # priorize the taglib1 package introduced in https://aur.archlinux.org/packages/taglib1
+    set(ENV{PKG_CONFIG_PATH} "/usr/lib/taglib1/lib/pkgconfig/:$ENV{PKG_CONFIG_PATH}")
```

Comment:
@Holzhaus Only these two lines are ARCH specific. The rest is required to have two version of taglib installed. We may remove it but on the other hand it is convenient for our contributors and does not really hurt.